### PR TITLE
Add team members

### PR DESCRIPTION
**Pull request by sharlak1**
added members for research, collaborators and media

### CODE REVIEW

File: src/styles/OurTeamPage.module.less [hunk 1]
```diff
@@ -37,6 +37,7 @@
   flex-direction: column;
   align-items: flex-start;
   flex-shrink: 0;
+  width: 70%;
```
Comment (IanSaucy) [inline src/styles/OurTeamPage.module.less:40]: Is there a reason why you added this?

**Review by IanSaucy — CHANGES_REQUESTED** [review]
Overall looks good! Could you swap out the placeholder image with the one attached? It's an SVG so you might need to modify the underlying component to support that. You can make the name shorter as well that the default on the file.
person_FILL1_wght400_GRAD200_opsz48

File: src/styles/OurTeamPage.module.less (hunk 1 above)
Comment (sharlak1) [reply src/styles/OurTeamPage.module.less:40]: To fix title content responsiveness

**Review by IanSaucy — APPROVED** [review]
lgtm

**Pull request merged**